common/constants.go

```diff
@@ -16,6 +16,17 @@ limitations under the License.
 
 package common
 
+var DefaultDatabaseInstanceName = "database_instance_name"
+
+var DefaultDatabaseNames = []string{"database_one", "database_two", "database_three"}
+
+var AllowedDatabaseTypes = map[string]bool{
+	"mysql":    true,
+	"postgres": true,
+	"mongodb":  true,
+	"mssql":    true,
+}
```

Comment:
Since these won't be used by anything other than the webhooks in the api/v1alpha1 package, can we move these out to another file or to the api package?